.github/workflows/python-app.yml

```diff
@@ -23,7 +23,16 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8
+        pip install flake8 pytest
```

Comment:
Pytest is already included in requirements-dev.txt

Reply:
Should flake8 go in there too?

Reply:
might as well add it in makes sense to

Reply:
cool - I'll remove it

Reply:
can we added flake8 to dequirements-dev.txt makes sense to @thomasridd

Reply:
why not - makes it more explicit about the version of the linter we are using too